Ext: Added a sortBy feature (#1685)
kyma-project/busola: docs/extensibility/display-widgets.md

ResourceList widgets render a list of Kubernetes resources. The ResourceList widgets should be used along with [related resources](resources.md#datasources-section).

#### Widget-specific parameters

- **children** optional field used to obtain custom columns. If not set, the configuration will be reused based on the existing resource list defined in Busola.
- **sort** - optional sort option. Array of objects with the following properties that allow to sort by it's value:
  - **path** - _[required]_ contains the path to the data used for the column.

>   - **default** - optional flag. If set to true, the list view is sorted by this value by default.

Reviewer: ```suggestion
  - **default** - optional flag. If set to `true`, the list view is sorted by this value by default.
```

>   - **compareFunction** - optional [JSONata](https://docs.jsonata.org/overview.html) compare function. It is required to use `$first` and `$second` variables when comparing two values. There is a special custom function `$compareStrings` used to compare two strings e.g. `$compareStrings($first, $second)`

Reviewer: ```suggestion
  - **compareFunction** - optional [JSONata](https://docs.jsonata.org/overview.html) compare function. It is required to use `$first` and `$second` variables when comparing two values. There is a special custom function `$compareStrings` used to compare two strings, for example, `$compareStrings($first, $second)`
```

ResourceList widgets render a list of Kubernetes resources. The ResourceList widgets should be used along with [related resources](resources.md#datasources-section).

#### Widget-specific parameters

- **children** optional field used to obtain custom columns. If not set, the configuration is reused based on the existing resource list defined in Busola.

> - **sort** - optional sort option. It's an array of objects that allow you to sort by the value from the given **source**.

Reviewer: ```suggestion
- **sort** - optional sort option. It's an array of objects that allows you to sort by the value from the given **source**.
```

>   - **source** - _[required]_ contains a [JSONata](https://docs.jsonata.org/overview.html) expression used to fetch data for the column. In its simplest form it's just the path to the value.

Reviewer: ```suggestion
  - **source** - _[required]_ contains a [JSONata](https://docs.jsonata.org/overview.html) expression used to fetch data for the column. In its simplest form, it's the path to the value.
```

#### Widget-specific parameters

- **collapsible** - an optional array of extra widgets to display as an extra collapsible section. Uses the same format as the **children** parameter.
- **sort** - optional sort option. If set to true, it allows to sort via this value. Defaults to false. It can also be set to an object with the following properties:
  - **default** - optional flag. If set to true, the list view is sorted by this value by default.

>   - **compareFunction** - optional [JSONata](https://docs.jsonata.org/overview.html) compare function. It is required to use `$first` and `$second` variables when comparing two values. There is a special custom function `$compareStrings` used to compare two strings e.g. `$compareStrings($first, $second)`

Reviewer: ```suggestion
  - **compareFunction** - optional [JSONata](https://docs.jsonata.org/overview.html) compare function. It is required to use `$first` and `$second` variables when comparing two values. There is a special custom function `$compareStrings` used to compare two strings, for example, `$compareStrings($first, $second)`
```
